WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -376,117 +409,167 @@ - (void)configureNoResultsView
     // Refresh the NoResultsView Properties
     self.noResultsView.titleText = self.noResultsTitleText;
 
-    // Only add and animate no results view if it isn't already
-    // in the table view
-    if (![self.noResultsView isDescendantOfView:self.tableView]) {
-        [self.tableView addSubviewWithFadeAnimation:self.noResultsView];
+    // Only add and animate no results view if it isn't already in the table view
+    if (![self.noResultsView isDescendantOfView:self.view]) {
+        [self.view addSubviewWithFadeAnimation:self.noResultsView];
     } else {
         [self.noResultsView centerInSuperview];
     }
 
-    [self.tableView sendSubviewToBack:self.noResultsView];
+    [self.view bringSubviewToFront:self.noResultsView];
 }
 
-- (NSString *)noResultsTitleText
-{
-    if (self.syncHelper.isSyncing) {
-        return NSLocalizedString(@"Fetching comments...", @"A brief prompt shown when the comment list is empty, letting the user know the app is currently fetching new comments.");
-    }
 
-    return NSLocalizedString(@"Be the first to leave a commment.", @"Message shown encouraging the user to leave a comment on a post in the reader.");
+#pragma mark - Autolayout Helpers
+
+- (void)updateViewConstraints
+{
```

Comment:
I'm wondering about this change. Seems like we might be doing a lot of extra work to tear down and rebuild constraints unnecessarily since the constraints aren't really changing.  Example: This method is called twice for every orientation change.  Is it addressing some other issue?